express/scripts/utils.js

```diff
@@ -102,6 +102,17 @@ export async function removeIrrelevantSections(area) {
   });
 }
 
+async function overrideMiloBlocks(area) {
+  if (!area) return;
+  area.querySelectorAll('main > div').forEach((section) => {
+    const columnBlock = section.querySelectorAll('div.columns');
```

Comment:
very good 👏 
you can make this even better though. The block class is always the very first class of the div. You should check for this because there might be blocks with a variant named column

Reply:
I reviewed the plain.html file and it seems not have `block` class on each block. Only the block name and parameters.